static/js/src/chart-data.js

```diff
@@ -450,7 +450,13 @@ export var kernelReleases2204 = [
   {
     startDate: new Date("2023-02-17T00:00:00"),
     endDate: new Date("2023-07-22T00:00:00"),
-    taskName: "Ubuntu 22.04.3 LTS (v5.19)",
+    taskName: "Ubuntu 22.04.2 LTS (v5.19)",
+    status: "LTS",
+  },
+  {
+    startDate: new Date("2023-08-16T00:00:00"),
+    endDate: new Date("2024-02-18T00:00:00"),
+    taskName: "Ubuntu 20.04.3 LTS (v6.2)",
```

Comment:
shouldn't it be 22.04.3 instead?

Reply:
That's right, there was an error in the spreadsheet. Updated